.github/workflows/heapster.yml

```diff
@@ -0,0 +1,96 @@
+name: SAWScript Heapster
+on:
+  push:
+    branches: [wip-heapster]
+  pull_request:
+    branches: [wip-heapster]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        ghc: ["8.6.5"]
+        continue-on-error: [false]
+        include:
+          - os: macos-latest
+            ghc: "8.6.5"
+            continue-on-error: true
+    runs-on: ${{ matrix.os }}
+    name: SAWScript - GHC v${{ matrix.ghc }} - ${{ matrix.os }}
+    continue-on-error: ${{ matrix.continue-on-error }}
+    env:
+      OCAML_VERSION: 4.09.1
+      # also change in setup-ocaml step based on value of opamVersion:
+      # https://github.com/avsm/setup-ocaml/blob/v1.1.2/src/installer.ts#L56
+      SETUP_OCAML_ACTION_VERSION: 1.1.2
+      COQBITS_VERSION: 1.0.0
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          git config --global url.https://github.com/.insteadOf git@github.com:
```

Comment:
This line enables checkout of the branch without having any authentication setup. The heapster-saw submodule is specified using the SSH form of the URL, which requires you to be authenticated to GitHub via an SSH key to clone the repository: https://github.com/GaloisInc/saw-script/blob/wip-heapster/.gitmodules#L54

This line instructs git to treat all remotes as HTTP, allowing the CI to clone the submodules without having any special SSH stuff setup.

Merging wip-heapster into master with the submodule using the SSH format could cause problems for users with workflows that aren't expecting to have to authenticate with SSH just to clone the repository and pull submodules. I can yank this line so that the CI will fail in this case if this was an accidental inclusion and you want to continue guarding against it (CI in master will fail if wip-heapster is merged since it doesn't have a corresponding line like this). If you want the CI to not guard against this type of thing then I can also create a one-line patch against master to add this, so that merging wip-heapster into master won't cause a CI failure.